### PR TITLE
Fix typo and grammer errors

### PR DESCRIPTION
**Pull request by LebronAl**

### CODE REVIEW

File: tinykv/doc_ss/lab1.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ The above concepts or abstractions are about the raft instance. The following co
 - `Peer`. A peer is a member of a raft group or a region, by default 3 replicas are used and a region will have 3 different peers. A peer will have a `RawNode` inside which contains a raft instance.
 - `raftWorker`. The worker to process all the client requests routed to different region leaders or leader peers.
 - `peerMsgHandler`. The delegate used to process the client requests for a specific leader peer.
-- `applyWorker`. After the proposed requests and the related logs are committed, correspond apply requests will be routed to the apply worker, then these logs will be applied to the state machine which is the `badger` storage engine in tinykv.
+- `applyWorker`. After the proposed requests and the related logs are committed, correspond apply requests will be routed to the applyWorker, then these logs will be applied to the state machine which is the `badger` storage engine in tinykv.
```
Comment (you06) [inline tinykv/doc_ss/lab1.md:108]:
```suggestion
- `applyWorker`. After the proposed requests and the related logs are committed, correspond apply requests will be routed to the `applyWorker`, then these logs will be applied to the state machine which is the `badger` storage engine in tinykv.

```

File: tinysql/doc_ss/lab4a-README-zh_CN.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 在这一章节，我们将实现从客户端输入 SQL 到数据写入分布式的 KV 数据库中的全链路。
 
-详细的资料可以参考文章 [TiDB 源码阅读系列文章（三）SQL 的一生](https://pingcap.com/blog-cn/tidb-source-code-reading-3)，此处仅做简要说明。
+详细地资料可以参考文章 [TiDB 源码阅读系列文章（三）SQL 的一生](https://pingcap.com/blog-cn/tidb-source-code-reading-3)，此处仅做简要说明。
```
Comment (you06) [inline tinysql/doc_ss/lab4a-README-zh_CN.md:7]: IMO, "详细的资料" is fine.

File: tinysql/doc_ss/lab4c-README-zh_CN.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ select id, val + 10 from t where val < 10;
 我们以这条 SQL 为例。首先它是一条 Select 语句，Select 会通过 `executor/table_reader.go` 中的 `TableReaderExecutor` 执行，但是这里并不是直接返回读取到的结果，需要将 val + 10 后的结果返回给客户端，因此还需要使用到 `executor/projection.go` 中的 `ProjectionExec` 来对 Select 的结果做一次运算处理。
 
 - 1\. `executor/builder.go`，因为数据处理的顺序是先通过 `SelectionExec` 获取数据再使用 `ProjectionExec` 进行计算处理，所以最外层的是 `ProjectionExec`，内层是 `TableReaderExecutor`。在 build 阶段，首先会执行 `executorBuilder.build` 中调用到 `executorBuilder.buildProjection` 函数，`ProjectionExec` 一定会对下层的结果进行处理，所以有 children，这里会递归的调用 `executorBuilder.build` 函数来 build 子 Executor。
-- 2\. `executor/table_reader.go`，`TableReaderExecutor` 的数据源是 `TableReaderExecutor.resultHandler`，最后会通过 `distsql/select_result.go` 中的 `SelectResult` 来执行。`SelectResult` 仅会从 TiKV 中获取所需要的数据来减少数据的传输量。具体的调用链路是 `TableReaderExecutor.Next` 调用 `tableResultHandler.nextChunk`，其中通过 `selectResult.Next` 方法（定义在 `SelectResult` 接口中）填充 Chunk。
+- 2\. `executor/table_reader.go`，`TableReaderExecutor` 的数据源是 `TableReaderExecutor.resultHandler`，最后会通过 `distsql/select_result.go` 中的 `SelectResult` 来执行。`SelectResult` 仅会从 TiKV 中获取所需要的数据来减少数据的传输量。具体地调用链路是 `TableReaderExecutor.Next` 调用 `tableResultHandler.nextChunk`，其中通过 `selectResult.Next` 方法（定义在 `SelectResult` 接口中）填充 Chunk。
```
Comment (you06) [inline tinysql/doc_ss/lab4c-README-zh_CN.md:14]: ditto, "具体的调用链路" is the suitable one.

**Review by you06 — APPROVED** [review]
Rest LGTM, thanks for the patch!

File: tinysql/doc_ss/lab4c-README-zh_CN.md (hunk 3 above)
Comment (LebronAl) [reply tinysql/doc_ss/lab4c-README-zh_CN.md]: ok

File: tinysql/doc_ss/lab4a-README-zh_CN.md (hunk 2 above)
Comment (LebronAl) [reply tinysql/doc_ss/lab4a-README-zh_CN.md]: ok

**Review by longfangsong — APPROVED** [review]

**Pull request merged**